.gitignore

```diff
@@ -8,3 +8,4 @@ node_modules/
 test-results.xml
 **/xcuserdata/*
 **/xcshareddata/*
+/dist
```

Comment:
not sure about the leading slash...
```suggestion
dist
```

Reply:
Leading slash is valid but np, will fix to match everything else

Reply:
Fixed